task/aws/resources/resource_launch_template.go

```diff
@@ -55,9 +55,11 @@ func (l *LaunchTemplate) Create(ctx context.Context) error {
 
 	size := l.Attributes.Size.Machine
 	sizes := map[string]string{
+		"s":       "t2.micro",
 		"m":       "m5.2xlarge",
 		"l":       "m5.8xlarge",
 		"xl":      "m5.16xlarge",
+		"s+t4":    "g4dn.xlarge",
 		"m+k80":   "p2.xlarge",
 		"l+k80":   "p2.8xlarge",
 		"xl+k80":  "p2.16xlarge",
```

Comment:
also, if none of these `short` -> `long` mapping succeed && the result still looks like a short string, we should log a warning mentioning https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/generic-machine-types

Reply:
I still believe that this kind of behavior, while convenient, is [not a priority](https://github.com/iterative/terraform-provider-iterative/pull/525#discussion_r856149156) for now. 🤔